tests/test_modeling_common.py

```diff
@@ -2272,6 +2272,56 @@ def test_no_super_init_config_and_model(self):
         for p1, p2 in zip(model.parameters(), new_model.parameters()):
             self.assertTrue(torch.equal(p1, p2))
 
+    @require_torch
```

Comment:
(nice to have)

Could it make sense to also test the function `save_and_shard_checkpoint` separately?